src/components/DropDownText/DropDownText.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Typography } from "@mui/material"
+import React, { useState } from 'react'
+import RenderIf from "../RenderIf/RenderIf"
+import { colors } from '../../constants/constants'
```

Comment:
With this you've created a concept of global colors, and that should probably be more of a style sheet. Constants, styles, and global styles are different. 

Additionally, constants should be `ALL_CAPS_WITH_UNDERSCORES_INSTEAD_OF_SPACES`

Reply:
I had a bit of a hard time converting over to a gobal CSS modules so i decided to just push this one out and add global styling later. ill make a ticket and add it to Jira.